table.cpp

```diff
@@ -95,11 +94,12 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   academic_integrity = ai;
   event = e;
 
-  if (reason != "") {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension due to "+reason+" on "+gID+"\" ";
-  } else {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension without specified reason on "+gID+"\" ";
-  }
+  if (event == "Extension" && reason != "") {
+      hoverText = "class=\"hoverable-cell\" data-hover-text=\"" + userName + " received a " + std::to_string(daysExtended) + " day extension due to " + reason + " on " + gID + "\" ";
```

Comment:
What I was suggesting is to move the hoverText inside the if statement below, which is line 109

Reply:
Updated in [c3528db](https://github.com/Submitty/RainbowGrades/pull/77/commits/c3528db46f9dd88b7d50f0bb5bc538d7efddc5b3), but functionality still needs to be tested.